src/Microsoft.Health.Blob/Registration/DefaultBlobDataStoreConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Blob.Configs;
+using Microsoft.Health.Blob.Features.Storage;
+
+namespace Microsoft.Health.Blob.Registration
+{
+    internal class DefaultBlobDataStoreConfiguration : IConfigureOptions<BlobDataStoreConfiguration>
```

Comment:
Futuremore, I'm thinking we probably could replace  `Action<T> configure` when configure with `IConfigureOptions` in the future.

Reply:
This class has been removed and its logic has been moved to a `PostConfigure` method invocation